andi/typeutils.py

```diff
@@ -19,14 +19,19 @@ def is_union(tp) -> bool:
 
 
 def get_union_args(tp) -> List:
-    """ Return a list of typing.Union args.
-    NoneType objects are converted to None for simplicity.
-    """
-    return _none_type_to_none(tp.__args__)
+    """ Return a list of typing.Union args. """
+    return list(tp.__args__)
 
 
-def _none_type_to_none(lst):
-    return [(el if el is not None.__class__ else None) for el in lst]
+def select_type(types, can_provide, bindings):
```

Comment:
currently this module provides typing-related utility functions which are not related to how andi works; what do you think about moving select_type to andi.py?

Reply:
done in https://github.com/scrapinghub/andi/pull/8

Reply:
:+1: